docs/root/configuration/other_features/_include/hyperscan_regex_engine.yaml

```diff
@@ -0,0 +1,48 @@
+static_resources:
```

Comment:
@phlax how do we ensure these don't bitrot?

Reply:
they are tested in ci both for validity and runnability